backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/VersionInputHandler.java

```diff
@@ -78,11 +79,15 @@ public VersionInputHandler(
    */
   public InternalFuture<Void> validateAndInsertVersionedInputs(
       String runId, VersioningEntry versioningEntry) {
-    if (versioningEntry != null) {
+    if (versioningEntry == null) {
```

Comment:
I'm pretty sure this will never be `null` because `getXYZ` in grpc returns the zero value of the object. Could you verify?

Reply:
Yes agree with you. it will never null if we will directly get it from proto but this is a PUBLIC method and if some one set null explicitly then it will throw null pointer so to protect null pointer i have added it for safe side.

Reply:
Fair